src/dealDashboard/dealDescription/dealDescription.scss

```diff
@@ -0,0 +1,5 @@
+.overview {
```

Comment:
Can you please put all of this css into a single container class?

Reply:
updated. I used the component itself as the container. No need for an additional element